tests/functional/authentication/02-suite2-stop-suite1.t

```diff
@@ -43,7 +42,7 @@ __FLOW_CONFIG__
 cylc register "${NAME2}" "${SUITE2_RUND}"
 cylc run --no-detach "${NAME1}" 1>'1.out' 2>&1 &
 SUITE_RUN_DIR="${SUITE1_RUND}" poll_suite_running
-run_ok "${TEST_NAME_BASE}" cylc run --no-detach "${NAME2}"
+run_ok "${TEST_NAME_BASE}" cylc run --no-detach --fragile "${NAME2}"
```

Comment:
(Let's just go with the long option, so no one has to look up `--fragile` to see what it actually means)